_inc/client/components/connect-button/index.jsx

```diff
@@ -139,7 +139,7 @@ export class ConnectButton extends React.Component {
 				{ ! this.props.isSiteConnected &&
 					<p className="jp-banner__tos-blurb">
 					{ __(
-						'By connecting your site you agree to our fascinating {{tosLink}}Terms of Service{{/tosLink}} and to {{shareDetailsLink}}share details{{/shareDetailsLink}} with WordPress.com',
+						'By clicking on the button below, you agree to our fascinating {{tosLink}}Terms of Service{{/tosLink}} and to {{shareDetailsLink}}share details{{/shareDetailsLink}} with WordPress.com.',
```

Comment:
In the interests of brevity, I'd remove "on" and just say "By clicking the button..."